org.gecko.vaadin.generator/bnd.bnd

```diff
@@ -1,5 +1,7 @@
 -buildpath: \
 	osgi.annotation;version='7.0.0',\
+	osgi.core;version='7.0.0',\
+	osgi.cmpn;version='7.0.0',\
```

Comment:
The generator is not real OSGi component. So we dont need to add unused and unnedded dependencies to the buildpath.

Reply:
Just like the `dependson` on issue - which caused error, and it was not possible to move forward without solving it, this one was a solution to a warning; screenshots attached: 

![Screenshot from 2021-09-10 12-26-48](https://user-images.githubusercontent.com/9023040/133089515-e3c692ca-e920-4658-b55d-1c857cf1b73e.png)
![Screenshot from 2021-09-10 12-36-08](https://user-images.githubusercontent.com/9023040/133089518-2a5cc55e-38f1-4105-9800-7f2e525129f9.png)